koko/riscv64.patch

```diff
@@ -0,0 +1,24 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -5,7 +5,7 @@ pkgver=23.04.0
+ pkgrel=1
+ pkgdesc='Image gallery application'
+ url='https://apps.kde.org/koko/'
+-arch=(x86_64 riscv64)
++arch=(x86_64)
```

Comment:
Is this committed by mistake? (We don't need the reversing `arch` change also here)

Reply:
Yes, it is committed by mistake :(